CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## v2.1.1 / 2021-07-28
+
+* [CHANGE] Replace multiListWatcher with independent listWatchers per namespace #1499
```

Comment:
These are changes not bug fixes from what I can tell, do we still want to include them in v2.1.1? Also the bug fixes seem not related to the 2.1 release, so maybe we need to instead cut 2.2? What do you think?

Reply:
I believe they are internal changes (that fix certain issues), so suitable for a patch release.
I would release v2.2 with kubernetes 1.22 support.

Reply:
That sounds good to me, thanks!

Reply:
I think for 1.9.x we can do that upon request and if someone volunteers doing the backports (so we are sure that there's a need). 
I believe for 2.0 we don't need to create a new release, 2.1 should work for those users as well.